controllers/hcpvaultsecretsapp_controller.go

```diff
@@ -288,6 +295,66 @@ func (r *HCPVaultSecretsAppReconciler) Reconcile(ctx context.Context, req ctrl.R
 	}, nil
 }
 
+func (r *HCPVaultSecretsAppReconciler) startShadowSecretCleanupRoutine(ctx context.Context) {
+	logger := log.FromContext(ctx)
```

Comment:
May want to add a name to this logger using WithName:

```suggestion
	logger := log.FromContext(ctx).WithName("startShadowSecretCleanupRoutine")
```